lib/Fhp/Action/SendSEPADirectDebit.php

```diff
@@ -166,9 +166,9 @@ protected function createRequest(BPD $bpd, ?UPD $upd)
 
             /* @var HIDMESv1 $hidxes */
             // Just always send the control sum
-            //if ($hidxes->getParameter()->summenfeldBenoetigt) {
+            // if ($hidxes->getParameter()->summenfeldBenoetigt) {
```

Comment:
Should this commented code just be removed entirely?

Reply:
I didnt work on SendSEPADirectDebit at all, so difficult for me to tell.

Reply:
> I didnt work on SendSEPADirectDebit at all

Then why is it part of this PR? Perhaps you accidentally ran the auto-formatter and would want to revert those changes? Same in MT535.php? (And then squash it all into one commit.)